### ISSUE TITLE
Show warnings generated by glslang

### ISSUE THREAD

**Issue by tomaka** [vulkano-rs/vulkano#117]
Once https://github.com/rust-lang/cargo/pull/2630 lands

**Comment by rukai** [comment]
glsl-to-spirv is no more. https://github.com/vulkano-rs/vulkano/pull/947

**Comment by nicokoch** [comment]
Could warnings be shown regardless?
Or are they shown already? Haven't really used vulkano recently.

**Comment by rukai** [comment]
Good point, shaderc makes it really easy to get. https://docs.rs/shaderc/0.3.11/shaderc/struct.CompilationArtifact.html#method.get_warning_messages

**Comment by Rua** [comment]
The feature linked by Tomaka isn't actually relevant here, because Vulkano-shaders is a procedural macro rather than a build script. Emitting warnings from a procedural macro is possible via the [proc_macro_error](https://docs.rs/proc-macro-error/latest/proc_macro_error/) crate, but this is not possible on stable Rust yet because it's blocked by [this feature](https://github.com/rust-lang/rust/issues/54140).
